src/components/Wiki/Wiki.tsx

```diff
@@ -78,6 +82,22 @@ class Wiki extends React.Component<wikiProps, wikiState>{
                 data['entries'] = data['entries'].sort();
                 this.setState({data: data as wikiData});
             })
+
+        fetch('http://localhost:8000/user?username='+cookies.get("username"),{
```

Comment:
Will want to replace this with the stored variable determining if this is local or prod.

Reply:
d9f7d8e 
ff28611